src/Widgets/Stats.php

```diff
@@ -24,10 +22,11 @@ public function render(): string
         if ($config) {
             if (substr_count($config, '-') === 3) {
                 $params = explode('-', $config);
-                $repository = $this->container->get($params[0]);
 
```

Comment:
https://symfony.com/doc/current/service_container/tags.html#the-index-by-indexattribute-option